tests/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:sid
```

Comment:
Oh, Debian testing isn't enough?

Reply:
That does not matter actually.

Reply:
(No big deal anyway, I guess)

Reply:
Running from testing would be better as it shields they package from transitions happening in Sid better in my opinion.